Readme.md

```diff
@@ -125,9 +125,22 @@ Custom HTML file:
 ```html
 $ echo \
   '<!doctype html>
-  <link rel="stylesheet" href="/style.css" />
-  <script src="/elm.js"></script>
-  ' > index.html
+<html>
+  <head>
```

Comment:
`<html>`, `<head>` and `<body>` are all optional in HTML5. While I often use them in real work for explicitness, I think we’re fine to leave them out here – to keep the example minimal.

Fine to leave the `<body>` though because of the [caveat](https://github.com/tomekwi/elm-live/#/caveats).